File: gateOne/DayOne/Java codes/TaskThree.java
import java.util.Scanner;
 public class TaskThree {
  public static void main(String[] args) {

	Scanner scan = new Scanner(System.in);
	
	double sum = 0;
	double average = 0;
	for(int count = 0;count < 10;count++) {
		System.out.println("Enter Number");
		int number = scan.nextInt();
		sum += number;
	}
	average = sum / 10;

	System.out.println(sum);
	System.out.printf("%.2f",average);
	
}



}

File: gateOne/DayOne/Java Script codes/taskThree.js
let sum = 0;
let average = 0;
let divisor = 10;

for(let count = 0;count < 10;count++) {
	const prompt = require ("prompt-sync")();
	let score = parseInt(prompt("Enter Number: "))	
	sum += score;
}

average = sum / divisor;

console.log(sum); 
console.log(average); 